packages/ckeditor-cloud-services-core/src/token/token.js

```diff
@@ -106,33 +106,53 @@ class Token {
 	 */
 	refreshToken() {
 		return this._refresh()
-			.then( value => this.set( 'value', value ) )
+			.then( value => {
+				this.set( 'value', value );
```

Comment:
Maybe we could throw an error if a value is empty or is the token format is incorrect. 

> A well-formed JWT consists of three concatenated Base64url-encoded strings, separated by dots (`.`):

Also, very often the endpoint of customers returns token in "" (`"token"`), so we could check it too. Of course, the token will be validated by the server but we could throw an error earlier. 

This is the only idea.

Reply:
I've added token validation.